cds-snc/covid-alert-app
# Check location service on Android

Resolves https://github.com/cds-snc/covid-alert-app/issues/891

## How to test
- Run
```
rm -rf node_modules/react-native-system-setting
yarn add react-native-system-setting@1.7.4
yarn
```
- Open the app, pass onboarding flow and enable EN. 
- Verify CovidAlert is active.
- Go to device settings, disable location.
- Back to the app, verify CovidAlert is inactive. 
- Turn on EN in the app, verify that location is enabled. 
- Turn off bluetooth. 
- Back to the app, verify CovidAlert is inactive.
- Turn on EN in the app, verify that bluetooth is enabled. 
- Keep the app in foreground, toggle Location off in notification bar (not system settings), verify that the app change status properly.
